crates/key-server/src/server.rs

```diff
@@ -106,6 +108,10 @@ struct FetchKeyRequest {
     enc_verification_key: ElgamalVerificationKey,
     request_signature: Ed25519Signature,
 
+    // Optional MVR name. If given and it resolves to the package from the ptb, this is the name
+    // used in the signed message.
+    mvr: Option<String>,
```

Comment:
maybe this should be part of cert? since it's set once and signed by the user for the rest of the session